include/sealib/compactarray.h

```diff
@@ -0,0 +1,48 @@
+#ifndef SEALIB_COMPACTARRAY_H_
+#define SEALIB_COMPACTARRAY_H_
+
+#include <math.h>
+#include <third-party/boost/include/boost/dynamic_bitset.hpp>
```

Comment:
Does <boost/dynamic_bitset> suffice?

Reply:
Not on my machine. There seems to be trouble with the CMake include path.

Reply:
You can not build it using the command line or does just your IDE does not recognise the include path?